popSimManu.tex

```diff
@@ -295,10 +314,12 @@ \section*{Results}
 extremely compact, and can be processed efficiently using the \tskit library
 \citep{kelleher2016efficient,kelleher2018efficient}. Currently,
 \stdpopsim uses the  \texttt{msprime} coalescent simulator \citep{kelleher2016efficient}
-as the default simulation engine. We plan to include other simulation
-engines to support scenarios that cannot be modelled under the coalescent framework,
-and have already implemented \texttt{SLiM} \citep{haller2019tree,haller2019slim} as
-an additional backend.
+as the default simulation engine. 
+We have implemented \texttt{SLiM} \citep{haller2019tree,haller2019slim} 
+as an alternative back-end, and plan to include other simulation
+engines to support scenarios that cannot be modelled under the coalescent framework.
+%ALG: previous version of the last sentence sounded kind of weird to me. But maybe
```

Comment:
this is a good suggestion

Reply:
Changed to
```
We have implemented \texttt{SLiM} \citep{haller2019tree,haller2019slim}
as an alternative backend,
to allow simulation of processes that cannot be modeled under the coalescent.
```
as I don't think we actually have plans for non-SLiM backends?